server/queue/queue-activemq/src/test/java/org/apache/james/queue/activemq/ActiveMQMailQueueBlobTest.java

```diff
@@ -69,7 +69,8 @@ public void setUp(BrokerService broker) {
         RawMailQueueItemDecoratorFactory mailQueueItemDecoratorFactory = new RawMailQueueItemDecoratorFactory();
         NoopMetricFactory metricFactory = new NoopMetricFactory();
         String queueName = BrokerExtension.generateRandomQueueName(broker);
-        mailQueue = new ActiveMQMailQueue(connectionFactory, mailQueueItemDecoratorFactory, queueName, USE_BLOB, metricFactory);
+        ActiveMQConsumerOptions consumerOptions = ActiveMQConsumerOptions.builder().prefetchSize(0).build();
```

Comment:
what about offering a `noPrefetch()` builder method ?

Reply:
and BTW it's never clear if 0 is "no" or "infinity" for example, so having a dedicated method is always better IMO

Reply:
I thought about that and had a feeling that introducing `noPrefetch()` method will be highly coupled with our issue.

Perfectly OK with that.